### PR TITLE
Index page for different versions

### LINKED ISSUE #702: Dashboard index page for different versions
Dashboard index page for different versions. After #697 and #662 is implemented.

The dashboard for a version will be at  `<groupId>/<artifactId>/<versions>/index.html`.

Index page would be at `<groupId>/<artifactId>/index.html`

### PR DESCRIPTION
**Pull request by suztomo**
Fixes #702 

```
~/cloud-opensource-java/dashboard$ mvn -B exec:java -Dexec.mainClass="com.google.cloud.tools.opensource.dashboard.DashboardMain"   -Dexec.arguments="-a com.google.cloud:libraries-bom"
...
```

image

### CODE REVIEW

File: dashboard/src/main/java/com/google/cloud/tools/opensource/dashboard/DashboardMain.java [hunk 1]
```diff
@@ -87,6 +87,8 @@
   public static final String TEST_NAME_GLOBAL_UPPER_BOUND = "Global Upper Bounds";
   public static final String TEST_NAME_DEPENDENCY_CONVERGENCE = "Dependency Convergence";
 
+  private static Configuration freemarkerConfiguration = configureFreemarker();
```
Comment (elharo) [inline dashboard/src/main/java/com/google/cloud/tools/opensource/dashboard/DashboardMain.java]: final?

File: dashboard/src/main/resources/templates/version_index.ftl [hunk 2]
```diff
@@ -0,0 +1,19 @@
+<!DOCTYPE html>
+<html lang="en-US">
+<#include "macros.ftl">
```
Comment (elharo) [inline dashboard/src/main/resources/templates/version_index.ftl]: do we need the macros for something?

File: dashboard/src/main/java/com/google/cloud/tools/opensource/dashboard/DashboardMain.java (hunk 1 above)
Comment (suztomo) [reply dashboard/src/main/java/com/google/cloud/tools/opensource/dashboard/DashboardMain.java]: Updated

File: dashboard/src/main/resources/templates/version_index.ftl (hunk 2 above)
Comment (suztomo) [reply dashboard/src/main/resources/templates/version_index.ftl]: Removed this line

**Pull request merged**